### PR TITLE
style: channel use struct

### PR DESCRIPTION
**Pull request by baerwang**

### CODE REVIEW

File: validator.go [hunk 1]
```diff
@@ -170,15 +170,15 @@ func (v *validator) ValidateHttpRequestWithPathItem(request *http.Request, pathI
 	reqBodyValidator := v.requestValidator
 
 	// create some channels to handle async validation
-	doneChan := make(chan bool)
+	doneChan := make(chan struct{})
```
Comment (daveshanley) [inline validator.go:173]: I don't understand why? we only need a bit, we don't need a whole struct?

**Comment by daveshanley** [comment]
Can you help me understand the value of this?

File: validator.go (hunk 1 above)
Comment (baerwang) [reply validator.go:173]: No memory consumption,`struct{}` is the smallest data type available in Go

**Review by daveshanley — APPROVED** [review]
LGTM!

**Pull request merged**